### ISSUE TITLE
Add an About Section

### ISSUE THREAD

**Issue by jimjkelly** [jimjkelly/missile-defense#6]
This should be static so as to be indexed properly in search engines.

**Comment by wslafoy** [comment]
Let me know how you want to be credited in the about section. I'll work on the about page after my research deadline on dec.12

**Comment by jimjkelly** [comment]
However "lead technical monkey" or whatever works.  :P

**Comment by wslafoy** [comment]
HA HA, WHAT IS TIME

I will do this next week!

**Comment by jimjkelly** [comment]
Ha, no worries.  I've been sloth slow at getting things on my end done, I apologize for that.

![gitscout-image-from-clipboard-1](http://static.gitscout.com/3f384ee75f9a5c48ff665173e5e55991c03cd4c15356b9a4db3281320c8fe982.png)

**Comment by wslafoy** [comment]
Would it be most efficient if we linked the main Missile Exchange tool to a Wordpress-powered blog for the about page and future posts? Sort of like how http://blog.nuclearsecrecy.com/ and https://nuclearsecrecy.com/nukemap/ are linked?

**Comment by jimjkelly** [comment]
Yeah that probably makes sense - although I'd be a bit careful about self-hosted wordpress.  They tend to get out of date and before you know it it's hacked.
